From 9769b49ac0e479c0a52c80b8c9112e2a48a508c0 Mon Sep 17 00:00:00 2001
From: Aikar <aikar@aikar.co>
Date: Fri, 12 Jul 2013 21:29:19 -0400
Subject: [PATCH] Entity Serialization

add method to spawn an entity at a location if its not in any world (invalid)
---
 src/main/java/org/bukkit/entity/Entity.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main/java/org/bukkit/entity/Entity.java b/src/main/java/org/bukkit/entity/Entity.java
index 72af4fa..7cbc4ef 100644
--- a/src/main/java/org/bukkit/entity/Entity.java
+++ b/src/main/java/org/bukkit/entity/Entity.java
@@ -16,7 +16,7 @@ import org.bukkit.event.player.PlayerTeleportEvent.TeleportCause;
  * Represents a base entity in the world
  */
 public interface Entity extends Metadatable {
-
+    public boolean spawnAt(Location loc); // EMC
     /**
      * Gets the entity's current position
      *
-- 
1.8.3.2

